### DISCUSSION TITLE
Add commit hash to review comment threads when review suggestions are accepted/merged

### DISCUSSION THREAD

**Discussion by 0xdevalias** [community/community#4197] [Pull Requests]
I originally raised this directly with GitHub support in 🔒 https://support.github.com/ticket/personal/0/1150627 (May 15th 2021), but wanted to include it here for wider visibility as well.

> Hey,
> 
> Just wanted to drop a little feature request. I know it may not got implemented in a timely manner, but wanted to get it on the radar at the very least.
> 
> Currently when we do code reviews, our convention/process is for the PR author to comment in the 'review comment thread' with the commit hash that they fixed the review comment in, then mark it as resolved. This makes it easier for the reviewer to quickly check that the changes made satisfy their request.
> 
> While I understand it would be hard to handle this automatically in a generalised way, my suggestion here is that it would be cool to handle this in an automated way when using GitHub's 'insert a suggestion' feature, when that suggestion has been accepted/merged through the UI.
> 
> When a suggestion is accepted/merged through the UI currently, the issue is marked as resolved already. So the additional request here would be that it also comments the commit hash it was fixed in in that 'review comment thread' (or otherwise shows the commit hash for where the suggestion was applied; it doesn't necessarily have to be a comment, so long as that contextual information is easily accessible from the relevant view comment thread)
> 
> Thanks for listening!

Which I received the following reply for:

> Thanks for writing in with this detailed suggestion! Your ticket was automatically routed to our Developer Support team but our Product team is better suited to help you out with this request. I've forwarded your ticket to the Product team who reads and evaluates all feature requests like this, however please note that team cannot guarantee a response to every submission.
> 
> This means the Developer Support team will no longer be able to see changes to the ticket, so if you have a follow up that requires further support, please create a new ticket. Thank you again for taking the time to let us know how we can make GitHub work better for you!

**Comment by alcere** [comment]
❤️  Thanks for your detailed suggestion!  It looks like a cool feature! I will create an issue in our internal track to ensure our team evaluates this idea. Thanks again, and please, keep the feedback coming!
